src/pages/student/categories/CategoryDetail/components/SubcategoryCard/index.js

```diff
@@ -16,20 +20,10 @@ const SubcategoryCard = ({ category }) => {
               : `/categories/${category.id}/quizzes`
           }
         >
-          <div className={style.cardContent}>
-            <table style={{ width: '100%' }}>
-              <tbody>
-                <tr>
-                  <td className={style.forSeccolum}>{category?.description}</td>
-                </tr>
-                <tr>
-                  <td id={style.listTable}>Available Quizzes:</td>
-                  <td className={style.forSeccolum}>
-                    {category?.total_quizzes ?? '0'}
-                  </td>
-                </tr>
-              </tbody>
-            </table>
+          <div>
+            <a id={style.Subtitle}>
```

Comment:
- [ ] Change \<a> tag to \<p> instead to avoid browser errors shown in console.
![Screenshot from 2021-11-22 11-40-38](https://user-images.githubusercontent.com/88467532/142798281-59ccecc8-3d01-4726-a556-24aeeecea5db.png)